bin/delete-empty-mirrors.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+const Storage = require('storj-service-storage-models');
+const mongoose = require('mongoose');
+const Config = require('../lib/config');
+
+program.version('0.0.1')
+  .option('-c, --config <path_to_config_file>', 'path to the config file')
+  .option('-u, --mongourl <mongo connection string>', 'mongo url connection string')
+  .parse(process.argv);
+
+
+async function deleteEmptyMirrors() {
+  if (!process.env.NODE_ENV) {
+    throw new Error('NODE_ENV is not set');
+  }
+  try {
+    let mongourl;
+    let deleteCount = 0;
+
+    if (program.mongourl) {
+      mongourl = program.mongourl;
+    }
+
+    const config = new Config(process.env.NODE_ENV, program.config);
+    const storage = new Storage(
+      mongourl || config.storage.mongoUrl,
+      config.storage.mongoOpts
+    );
+
+    const Mirror = storage.models.Mirror;
+    const Shard = storage.models.Shard;
+
+    const cursor = Mirror.find()
+      .sort({
+        '_id': 1
+      })
+      .cursor();
+
+    const chunkOfMirrors = [];
+    const chunkSize = 5;
+
```

Comment:
Better print delete**d**Count on an interval too, with `setInterval()` and clean the timer at the end of the process

Reply:
good idea, added